packages/gatsby/src/query/graphql-runner.ts

```diff
@@ -194,7 +194,7 @@ export class GraphQLRunner {
               rootValue: context,
               contextValue: withResolverContext({
                 schema,
-                schemaComposer: schemaCustomization.composer,
+                schemaComposer: schemaCustomization.composer!,
```

Comment:
Can we avoid using non-null assertions if at all possible. If it's not really nullable then the type should be updated. If it _is_ nullable, then there should be a runtime check.

Reply:
The main issue is that we start the initialState with `null`, that's the reason why I added it as a type. I feel I should update it there to a better default, what do you think?

Reply:
Better default is likely the way to go. Let me look closer

Reply:
Tried to go with a better default. It had test issues. But I decided to go with null and add runtime checks. Since the value is added _later_ it's better to handle this way so that we can predictably write code against if the composer is available or not.